bin/hipify-perl

```diff
@@ -1801,6 +1803,9 @@ sub simpleSubstitutions {
     $ft{'define'} += s/\bcudaTextureType3D\b/hipTextureType3D/g;
     $ft{'define'} += s/\bcudaTextureTypeCubemap\b/hipTextureTypeCubemap/g;
     $ft{'define'} += s/\bcudaTextureTypeCubemapLayered\b/hipTextureTypeCubemapLayered/g;
+    $ft{'define'} += s/\bcudaOccupancyDefault\b/hipOccupancyDefault/g;
+    $ft{'define'} += s/\bcudaCooperativeLaunchMultiDeviceNoPreSync\b/hipCooperativeLaunchMultiDeviceNoPreSync/g;
+    $ft{'define'} += s/\bcudaCooperativeLaunchMultiDeviceNoPostSync\b/hipCooperativeLaunchMultiDeviceNoPostSync/g;
```

Comment:
It looks like hipify-perl changes were obtained by hand. Please, generate hipify-perl script by `hipify-clang -perl` and update the diff. hipify-clang should be built against your changes beforehand.

Reply:
generated the new hipify-perl